engines/director/frame.cpp

```diff
@@ -923,11 +923,10 @@ void readSpriteDataD5(Common::SeekableReadStreamEndian &stream, Sprite &sprite,
 			break;
 		case 2:
 			if (sprite._puppet) {
-				stream.skip(4);
+				stream.readSint16();
```

Comment:
this is 2 bytes instead of 4. Is this intentional?

Reply:
The change look good (because if you read 4 bytes, you never go in the case 4 below).